.github/workflows/main.yml

```diff
@@ -15,33 +15,55 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: 'Setup node'
+      - name: "Setup node"
         uses: actions/setup-node@v4
 
-      - name: 'Install vsce'
+      - name: "Install vsce"
         run: npm install -g @vscode/vsce
 
-      - name: 'Install esbuild'
+      - name: "Install esbuild"
         run: npm install esbuild
 
-      - name: 'Build extension'
+      - name: "Build extension"
         run: npm install; vsce package
 
       - uses: actions/upload-artifact@v4
         with:
           name: artifacts.zip
           path: imandrax-*.vsix
 
+  test:
```

Comment:
Thanks so much for figuring all this out! These tests will be super useful in the future!